documentation/admin.md

```diff
@@ -44,3 +48,11 @@ reload: true
 ```
 in its content
 
+What version is deployed can be checked with
+```
+curl -v localhost:8080/perseo-core/version.jsp
```

Comment:
Would it be possible to have the URL without '.jsp' at the end? To align with how the operation works in Orion, Cygnus, etc.